src/sentry/static/sentry/app/components/selectInput.jsx

```diff
@@ -29,6 +29,17 @@ const SelectInput = React.createClass({
 
   componentDidMount() {
     this.create();
+    /*below is a hack for a bug in edge related to form submitting.
+    see: https://github.com/facebook/react/issues/7655
+    (@maxbittker)*/
+    let selectedIndex = this.refs.select.selectedIndex;
+    if (this.refs.select && selectedIndex >= 0) {
```

Comment:
Should the check for `this.refs.select` occur earlier? Because you access a property of it (`selectedIndex`) the line before (L35). If `select` is truly falsy, then L35 will blow up because it cannot access property `selectedIndex` of null/undefined/0/etc.